README.md

```diff
@@ -33,6 +34,7 @@ Use:
 # contributor #
 > - 👤 Moise Rushanika
 
```

Comment:
Your Readme looks great :+1: 
However, there's one more thing, and that is the `Getting Started` section. This section is important for users to be able to set up your project locally on their PC. This [template](https://github.com/microverseinc/readme-template) might help